tests/mock_api/test_continuous_mastering.py

```diff
@@ -63,3 +63,62 @@ def test_continuous_mastering():
 
     clause1 = project.estimate_pairs().clause_estimates["clause1"]
     assert clause1["generatedPairCount"] == "25"
+
+
+@responses.activate
+def test_rcwd():
```

Comment:
Shorthand variable names are fine when (1) the context is clear and (2) you will need to type them more than 1x. But here it defines the name of this test, so we should opt for a more descriptive name:

```suggestion
def test_record_clusters_with_data():
```

That way, if this test fails, contributors aren't left scratching their heads as to what `rcwd` means